wurst/network/NetworkConfig.wurst

```diff
@@ -18,4 +18,4 @@ package NetworkConfig
 	Enables or disables various runtime safety checks in the library.
 	Do not disable this unless you are 100% sure your code works correctly.
 **/
-@configurable public constant SAFETY_CHECKS_ENABLED = true
\ No newline at end of file
+@configurable public constant NW_SAFETY_CHECKS_ENABLED = true
```

Comment:
why rename?? if ppl config this it's breaking change.

Reply:
conflict with buffer's configurable of the same name